src/pages/Newsletter/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+//import Input from '../../components/Input'
```

Comment:
Você não precisa comentar código se não for pra explicar algo. Pode remover essas duas linhas :)

Reply:
Feito no último commit!